### PR TITLE
Update readme validators table

### PR DESCRIPTION
**Pull request by academo**
Updates the readme to include recently added validators and sort them alphabetically

### CODE REVIEW

**Review by xnyo — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -193,22 +193,27 @@ The tool runs a series of analyzers to ensure submitted plugins are following be
...
+| Backend Binary / `backendbinary` | Checks that binary exists for plugins declaring a backend in plugin.json | None |
 | Backend Debug / `backenddebug`                    | Checks that the standalone debug files for backend plugins are not present.                                                       | None                                                                  |
 | Binary Permissions / `binarypermissions`          | For datasources and apps with binaries, this ensures the plugin can run when extracted on a system.                               | None                                                                  |
 | Broken Links / `brokenlinks`                      | Detects if any URL doesn't resolve to a valid location.                                                                          | None                                                                  |
+| Checksum / `checksum` | Validates that the passed checksum (as a validator arg) is the one calculated from the archive file | `checksum` |
```
Comment (sympatheticmoose) [inline README.md:200]:
```suggestion
| Checksum / `checksum` | Validates that the passed checksum (as a validator arg) is the one calculated from the archive file. | `checksum` | 
```
Would you do this locally? I can't see where we describe how you would provide this when running outside of the plugin submission form

File: README.md [hunk 2]
```diff
@@ -193,22 +193,27 @@ The tool runs a series of analyzers to ensure submitted plugins are following be
...
 | Binary Permissions / `binarypermissions`          | For datasources and apps with binaries, this ensures the plugin can run when extracted on a system.                               | None                                                                  |
 | Broken Links / `brokenlinks`                      | Detects if any URL doesn't resolve to a valid location.                                                                          | None                                                                  |
+| Checksum / `checksum` | Validates that the passed checksum (as a validator arg) is the one calculated from the archive file | `checksum` | 
 | Code Rules / `coderules`                          | Checks for forbidden access to environment variables, file system or use of `syscall` module.                                     | [semgrep](https://github.com/returntocorp/semgrep), `sourceCodeUri`   |
+| Discoverability / `discoverability` | Warns about missing keywords and description that are used for plugin indexing in the catalogue | None|
```
Comment (sympatheticmoose) [inline README.md:202]:
```suggestion
| Discoverability / `discoverability` | Warns about missing keywords and description that are used for plugin indexing in the catalog. | None |
```

File: README.md [hunk 3]
```diff
@@ -193,22 +193,27 @@ The tool runs a series of analyzers to ensure submitted plugins are following be
...
+| Developer Jargon / `jargon`                       | Generally discourages use of code jargon in the documentation.                                                                     | None                                                                |
 | JS Source Map / `jssourcemap`                     | Checks for required `module.js.map` file(s) in archive.                                                                             | `sourceCodeUri`                                                       |
-| Logos / `logos`                                   | Detects whether the plugin includes small and large logos to display in the Plugin catalog                                        | none                                                                  |
-| Developer Jargon / `jargon`                       | Generally discourages use of code jargon in the documentation.                                                                     | None                                                                  |
+| Legacy Grafana Toolkit usage / `legacybuilder` | Detects the usage of the not longer supported Grafana Toolkit | None |
```
Comment (sympatheticmoose) [inline README.md:207]:
```suggestion
| Legacy Grafana Toolkit usage / `legacybuilder` | Detects the usage of the not longer supported Grafana Toolkit. | None |
```

File: README.md [hunk 4]
```diff
@@ -193,22 +193,27 @@ The tool runs a series of analyzers to ensure submitted plugins are following be
...
 | Legacy Platform / `legacyplatform`                | Detects use of Angular which is deprecated.                                                                                       | None                                                                  |
 | License Type / `license`                          | Checks the declared license is one of: BSD, MIT, Apache 2.0, LGPL3, GPL3, AGPL3.                                                  | None                                                                  |
+| Logos / `logos`                                   | Detects whether the plugin includes small and large logos to display in the Plugin catalog                                        | none                                                                  |
```
Comment (sympatheticmoose) [inline README.md:210]:
```suggestion
| Logos / `logos`                                   | Detects whether the plugin includes small and large logos to display in the plugin catalog.                                        | none                                                                  |
```

**Review by sympatheticmoose — APPROVED** [review]
Some nits for consistency and one suggestion on binary description

**Pull request merged**